waf.go

```diff
@@ -112,14 +112,11 @@ func Load() (ok bool, err error) {
 	return wafLib != nil, wafErr
 }
 
-// Health returns an error when this package is not in a usable state describing
-// the reason why.
-func Health() error {
-	ok, err := Load()
-	if !ok {
-		return err
-	}
-	return nil
+// SupportsTarget returns true and a nil error when the target host environment
+// is not supported, such as with a non-supported operating-system.
+// Otherwise, it returns false along with an error detailing why.
```

Comment:
```suggestion
// SupportsTarget returns true and a nil error when the target host environment
// is supported by this package and can be further used.
// Otherwise, it returns false along with an error detailing why.
```